Templates/BlobTrigger-ImageResize-CSharp/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Image resizing with a blob trigger - C#",
```

Comment:
This name doesn't match the naming scheme of all the other templates - this name looks more like a description. Wondering how well it will display in the UI.